Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_sr.yml

```diff
@@ -0,0 +1,272 @@
+# Station Representative Forms
+# Sources: https://discord.com/channels/1123826877245694004/1182366744266932224/1182366744266932224
+#          https://docs.google.com/document/d/1jDX9numj-VHKBJ2FiiEo1GIlHESffOUIHIIXaqhEnls/edit?pli=1#heading=h.ru93r8n0fb7m
+
+# Filled Folder
+- type: entity
+  id: BoxFolderSr
+  parent: BoxFolderBase
+  name: SR folder
+  description: A folder filled with paperwork templates.
+  suffix: Forms
+  components:
+  - type: Sprite
+    layers:
+    - state: folder-colormap
+      color: "#1f6626ff"
+    - state: folder-base
+  - type: StorageFill
+    contents:
+    - id: PaperWrittenFrontierFormEmployment
+    - id: PaperWrittenFrontierFormEmployeePayHistory
+    - id: PaperWrittenFrontierFormFundRequisition
+    - id: PaperWrittenFrontierFormShuttleTitleChange
+    - id: PaperWrittenFrontierFormAuthorization
+    - id: PaperWrittenFrontierFormShuttleDerelict
+    - id: PaperWrittenFrontierFormContrabandTurnover
+    - id: PaperWrittenFrontierFormContrabandPermit
+    - id: Paper
+      prob: 0.5
+    - id: Paper
+      prob: 0.5
+
+# Forms
+- type: entity # Based on form made by Willzile1 (Jon-Gah)
+  id: PaperWrittenFrontierFormEmployment
+  name: employment contract
```

Comment:
Thoughts on capitalizing these names to match the title?  I know it's just a form, but it's still titled.

Reply:
None. If you think it'll be better that way, then sure.